### PR TITLE
OTNS Integration: Display Test Title

### PR DESCRIPTION
**Pull request by ctan-g**
This PR works alongside https://github.com/openthread/ot-ns/pull/52 to display title of currently running Silk test case name.
Screenshot from 2020-07-20 14-01-48

### CODE REVIEW

File: silk/tools/otns_manager.py [hunk 1]
```diff
@@ -87,6 +87,16 @@ def __init__(self, server_addr: str, logger: logging.Logger):
     self.stub = visualize_grpc_pb2_grpc.VisualizeGrpcServiceStub(self.channel)
     self.logger = logger
 
+  def send_title(self, title: str):
```
Comment (simonlingoogle) [inline silk/tools/otns_manager.py]:
```suggestion
  def set_title(self, title: str):
```
be consistent with proto?

File: silk/tools/otns_manager.py [hunk 2]
```diff
@@ -87,6 +87,16 @@ def __init__(self, server_addr: str, logger: logging.Logger):
...
+    """Send test title to OTNS.
+
+    Args:
+        title (str): test title.
```
Comment (simonlingoogle) [inline silk/tools/otns_manager.py]: Why not also support setting position (x, y) and font_size?

Comment (ctan-g) [reply silk/tools/otns_manager.py]: Sure thing. I wasn't sure how much Silk needs to care about the visualization details of the title, but I can give it the same default as OTNS.

**Review by simonlingoogle — APPROVED** [review]
LGTM 👍 
But we need to wait for https://github.com/openthread/ot-ns/pull/52 to be merged.

**Review by qilinkang — APPROVED** [review]
LGTM

**Review by jwhui — APPROVED** [review]

**Pull request merged**